README.md

```diff
@@ -1,26 +1,30 @@
-# gitflow-incremental-builder
+# gitflow-incremental-builder (GIB)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.vackosar.gitflowincrementalbuilder/gitflow-incremental-builder/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.vackosar.gitflowincrementalbuilder/gitflow-incremental-builder)
-
 ![Travis CI](https://travis-ci.org/vackosar/gitflow-incremental-builder.svg?branch=master)
-
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/32140688527a49deb3bd45b8f3be4acf)](https://www.codacy.com/app/vackosar/gitflow-incremental-builder?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=vackosar/gitflow-incremental-builder&amp;utm_campaign=Badge_Grade)
 
 A maven extension for incremental building of multi-module projects when using [feature branches (Git Flow)](http://nvie.com/posts/a-successful-git-branching-model/).
 Builds or tests only changed maven modules compared to reference branch in Git (e.g. origin/develop) and all their dependents.
 
+This extension is **not limited to Git Flow setups!** The [extensive configuration options](#configuration) provide support many other branch setups and/or use cases. 
+
 ## Table of Contents
 
 - [Usage](#usage)
-  * [Disable in IDE](#disable-in-ide)
+  - [Disable in IDE](#disable-in-ide)
+
 - [Example](#example)
 - [Configuration](#configuration)
-  * [gib.enabled](#gibenabled)
-  * [gib.disableBranchComparison](#gibdisablebranchcomparison)
-  * [gib.uncommited](#gibuncommited)
-  * [gib.untracked](#gibuntracked)
-  * [gib.skipTestsForNotImpactedModules](#gibskiptestsfornotimpactedmodules)
-  * [gib.argsForNotImpactedModules](#gibargsfornotimpactedmodules)
-  * [gib.forceBuildModules](#gibforcebuildmodules)
+  - [gib.enabled](#gibenabled)
+  - [gib.disableBranchComparison](#gibdisablebranchcomparison)
+  - [gib.uncommited](#gibuncommited)
+  - [gib.untracked](#gibuntracked)
+  - [gib.skipTestsForNotImpactedModules](#gibskiptestsfornotimpactedmodules)
+  - [gib.argsForNotImpactedModules](#gibargsfornotimpactedmodules)
+  - [gib.forceBuildModules](#gibforcebuildmodules)
+
+-   [Requirements](#requirements)
```

Comment:
This was just a test to see whether Codacy _really_ wants to have _three_ spaces between the bullet point and the content and this indeed removed the violation message!
See also: https://app.codacy.com/app/gitflow-incremental-builder/gitflow-incremental-builder/file/28858643410/issues/source?bid=10241438&fileBranchId=10805317#l15
The `space` seems what we want but I could not figure out how to set this in Codacy...

Reply:
Update: I found the setting for `space`! Testing this now...

Reply:
Fixed:
![Codacy RemarkLint setting](https://user-images.githubusercontent.com/22860528/51442918-5ffb4480-1ce2-11e9-923c-a6a70603b538.png)